### ISSUE TITLE
No Underline or chevron is provided for the "Learn more in our Docs" link to differentiate it from normal text: A11y_Awesome AZD_Template Library_ Usability

### ISSUE THREAD

**Issue by haileyhuber8** [Azure/awesome-azd#695]
GIT Hub Tags:
#A11yTCS; #A11ySev4; #Azure Developer CLI; #DesktopWeb; #A11yUsable; #P3_Awesome-AZD_Nov2024; #FTP; #Chromium Edge; #Win11; #CELA_P3; #WCP;

Environment and OS details:
Application name: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge version: Version 130.0.2849.52 (Official build) (64-bit)

Repro Steps:
Launch the url https://azure.github.io/awesome-azd/ screen will get open.
Tab till "Learn More in our Docs" link.
Observe that Underline or chevron is provided for the "Learn more in our Docs" link to differentiate it from normal text or not.

Actual behavior:
No Underline or chevron is provided for the "Learn more in our Docs" link to differentiate it from normal text

Expected behavior:
Underline or chevron should be provided for the "Learn more in our Docs" link to differentiate it from normal text.

User Impact:
Color blind yours will face difficulty in differentiating the links from normal text if no underline or chevron provided for the links.

Attachment:

<img width="1033" height="523" alt="Image" src="https://github.com/user-attachments/assets/f368f3ca-e5ed-4b25-bf9d-683e682be034" />
